proto/config.go

```diff
@@ -125,14 +125,14 @@ func (r *RangeDescriptor) ContainsKeyRange(start, end []byte) bool {
 }
 
 // FindReplica returns the replica which matches the specified store
-// ID. Panic in the event that no replica matches.
+// ID. Returns nil in the event that no replica matches.
 func (r *RangeDescriptor) FindReplica(storeID StoreID) *Replica {
```

Comment:
I needed another version of this in my `ds_attr` branch. You'll see when you rebase.